### PR TITLE
Layout Template : Header, Footer v2.0

### PR DESCRIPTION
**Pull request by at-vudang95**

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -2,10 +2,10 @@ import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
 
 import { AppComponent } from './app.component';
-
+import { TEMPLATECOMPONENTS } from './component/template';
```
Comment (ghost) [inline src/app/app.module.ts]: Should you define `TemplateComponents` for TEMPLATECOMPONENTS

File: src/app/component/template/breadcrumbs/breadcrumbs.component.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+        <li class="active" itemprop="itemListElement" itemscope="" itemtype="http://schema.org/ListItem">
+          <span itemprop="name">Tất cả sản phẩm</span>
+        </li>
+
+
+      </ol>
+    </nav>
+  </div>
+</div>
```
Comment (ghost) [inline src/app/component/template/breadcrumbs/breadcrumbs.component.html]: Remove line NULL for clean code.

File: src/app/component/template/footer/footer.component.css [hunk 3]
```diff
@@ -0,0 +1,6 @@
+a {
+  color: #acacac;
+}
+a:hover {
+  color: #fff;
+}
```
Comment (ghost) [inline src/app/component/template/footer/footer.component.css:6]: This CSS will be apply all entities ```a``` in this project.

File: src/app/component/template/footer/footer.component.html [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+      </div>
+    </section>
+    <section id="footernav" class="footer-nav">
+      <div class="container">
+        <div class="footer-bottom-wrap">
+          <div id="powered" class="powered ">
+            <!--<div class="copyright">-->
+              <!--<p class="text-center float-left">© Bản quyền thuộc về <a href="http://apollotheme.com/">Intern AT PHP</a>.</p>-->
+              <!--<p class="provide-by">Cung cấp bởi <a href="#" title="">Intern PHP</a></p>-->
+            <!--</div>-->
```
Comment (ghost) [inline src/app/component/template/footer/footer.component.html]: Pls remove if you don't used it.

File: src/app/component/template/header/header.component.html [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+</div>
+</div>
+</div>
+</div>
+</div>
+</div>
+<!-- end header main -->
+</section>
+</header>
+<!-- end header -->
```
Comment (ghost) [inline src/app/component/template/header/header.component.html]: Beautiful source code for this file.

File: src/app/component/template/footer/footer.component.css (hunk 3 above)
Comment (at-vudang95) [reply src/app/component/template/footer/footer.component.css:6]: It only all entities a in component footer

File: src/app/app.module.ts [hunk 6]
```diff
@@ -1,16 +1,17 @@
-import { BrowserModule } from '@angular/platform-browser';
-import { NgModule } from '@angular/core';
-
-import { AppComponent } from './app.component';
-
+import {BrowserModule} from '@angular/platform-browser';
+import {NgModule} from '@angular/core';
+import {AppComponent} from './app.component';
+import {TemplateComponents} from './component/template';
```
Comment (ghost) [inline src/app/app.module.ts]: I think you should change ```TemplateComponents``` to ```TemplateComponent```

**Review by ghost — APPROVED** [review]

**Pull request merged**